roles/core/time/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
My bad, the stack does not use roles defaults: https://github.com/bluebanquise/bluebanquise/pull/163#issuecomment-613333223
Please remove this file. Sorry for the extra work. :-(